pkg/virt-controller/services/template.go

```diff
@@ -951,6 +982,9 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, t
 			"--hook-sidecars", strconv.Itoa(len(requestedHookSidecarList)),
 			"--less-pvc-space-toleration", strconv.Itoa(lessPVCSpaceToleration),
 			"--ovmf-path", ovmfPath,
+		}...)
+		if nonRoot {
+			command = append(command, "--run-as-nonroot")
 		}
```

Comment:
This code segment, there are 2 IF blocks. It would be great If assign the common options, `--qemu-timeout .... --ovmf-path ...` to a variable and use one IF block.